django_x509/base/models.py

```diff
@@ -262,8 +271,12 @@ def _generate(self):
         cert.set_pubkey(key)
         cert = self._add_extensions(cert)
         cert.sign(issuer_key, str(self.digest))
+        key_args = (crypto.FILETYPE_PEM, key)
+        key_kwargs = {}
+        if self.passphrase:
+            key_kwargs['passphrase'] = self.passphrase.encode('utf-8')
```

Comment:
I have tested the generation of a new object with a passphrase and the dumped private key in the web UI is not encrypted, while digging [I found out we need to specify the cipher keword argument](https://github.com/pyca/pyopenssl/issues/688#issuecomment-325827337) too:
```python
key_kwargs['cipher'] = 'DES-EDE3-CBC'
```
Once you add that, in the web UI of a passphrase protected object you will see something like:
```
-----BEGIN ENCRYPTED PRIVATE KEY-----
MIIFHDBOBgkqhkiG9w0BBQ0wQTApBgkqhkiG9w0BBQwwHAQIecKG+xlkaOQCAggA
MAwGCCqGSIb3DQIJBQAwFAYIKoZIhvcNAwcECE3tJurNzsL7BIIEyE7KPV1A+ABB
rQs/pb4sEBhFJGgYoAnrwJtW1eOZ0MQI5Dytv7MgnCkhb9CoAeZldc1jnW59bqzI
gEM3btEHeZPpFITsvewt9kXQqTpfArlNzq60o/KGXxbocNyOAethLD5PBiVL/Zuy
VZAhLxaWkmQXiGDsOi+nSUDfvtGflBgAJM6CCX7/Q2SGwszg29XI3ziCmKPMIQpf
xAuWBIf2jp9kgIjfPBpBqg6GgRDcHW1BrgJjFGuht8fgv68vbia1syVJ4PADHC3n
PVf867w5CRKOyKh/4fqyw0MRdmZlxtiov6ZatXntDos8fPokkilKMUSfE5vWK4AF
3o3Y4COIsJasge8DS+joDnVKOh1HarX4x9NRowPAUgs9713UqauQcclS71eYRLOv
k6/p3Xhl8SGmbJVWjrjuc1fdzloF/PWOR3mroNQp1mnKTdxh8XYYtPQzefc1Y9N2
veXtmL94NE1/w0qqS5yx3Ah1KIv5bSr9JxFnaTJU7l/PCMU7Bejdjnk75nK+NlAf
0XwJkSgx84DfcLd57t64420TyedbViwlvjIAjYOKxPWSCHzzl2TRYcpKVEm3myFv
wcOn7zi0V0gVCSS16GUx6DGpf2JmfShoOVZKfMu+pCJsqaL03w3uUiNSJISQCI59
ItYY39XpSYqmwcFOGxSykxIBF+q8mAHaSx9WMpbAL0BKoQ8ZDEkogsEjKhzKYcj5
ZX3SB/3/BrtCiSsnLM9LUETDfwhJBPuUix2lUHksedoxVBtPS4+960SgFmuAwrIc
/3/7La/NjReuPqIoeZPsMfYa8NgYqSlDIQWaJk/NdHVV5VDFd4uvyB/2cDv1TMER
a3WTfvJDr0Li4ym9SOA6diQ+/CK/bCAlqbqcTUV6+rl+zL1RobFf0+/xUalX9FiS
uwX87Q7ZKweCvV+Xe2/wHx1GjBSBIDfoFx8MTgSGhguXYfc4hn3yBdDh4nKqqtp/
cX0TWifYIBNTyTkHoC+SAdqTOomQymr6ZZ69+54K6VpOHYBtt6CQJUleRLK7ZNhO
zhpMVFqzn5uPgeeAX6V4AuXOOLxfREpWsdFufK5ceVmp5IqhX+SHfIrBywz8JWaL
0W3oX9hizSGRl3wteYUA/cfZK76AgSZISBbtrsWUigPtknakxrOc3i+dVnVCeAhB
n7jNSX28+ONbgmwlHH5QYBtJMjFV71qWT1h06wI3lf3ozasjYgMiY3B8iK6zZHxU
lX60n/5AFZG9GyUHkWCCaJMmaFDkS9T0nFx0/ARgIrokSkmnlqsvG8jfCSd4Rp0S
vRQv1uofN/JwoMkec7O8EOtg3O4zK0GRNfXPhiB3H4KoSJGOnCy0/WCCYRJqTuly
9GRbxnWFBX1xhFkxDnK9hyRF/iKlyzoBPZKPq5sNTcwpc5mmTq/zEqQPZPkOEDBG
svqS2v6BKzTLEY2fwq7IOAFg/j4CgdufBh6avu44f/zH40thKfGYvTJsYNrMM/1I
VmCAJi6NmJ/yDik6VTfZhPTOd4pxzs5jIFssU0+HLNcWYZgZHJEQ2wcmgNl6ZSE8
jP5C0p0yM1tRkINipiugpaXWj9SfaRcRF6ndXUebvugUJlfwJQ4D1ljmqx8RYRWj
5g1TdagGLt14fipdgSTsmA==
-----END ENCRYPTED PRIVATE KEY-----
```